packages/yum/groonga.spec.in

```diff
@@ -45,7 +45,7 @@ on relational data model.
 Summary:	Runtime libraries for Groonga
 Group:		System Environment/Libraries
 License:	LGPLv2 and (MIT or GPLv2)
-Requires:	arrow-libs = @APACHE_ARROW_VERSION@
+Requires:	arrow-libs = @APACHE_ARROW_VERSION@-%{release}
```

Comment:
`arrow-libs` doesn't use the same release version as ours.

```suggestion
Requires:	arrow-libs = @APACHE_ARROW_VERSION@-1%{?dist}
```

We should change `BuildRequires:	arrow-devel = @APACHE_ARROW_VERSION@` too.